src/plotman/_tests/plot_util_test.py

```diff
@@ -45,15 +45,15 @@ def test_columns() -> None:
               [ 1 ],
               [ 2 ] ] )
 
-def test_list_k32_plots(fs: pyfakefs.fake_filesystem.FakeFilesystem) -> None:
+def test_list_k_plots(fs: pyfakefs.fake_filesystem.FakeFilesystem) -> None:
```

Comment:
Could you add test files for another size or two?

```suggestion
def test_list_plots(fs: pyfakefs.fake_filesystem.FakeFilesystem) -> None:
```